src/utils/helpers/date/__spec__.js

```diff
@@ -1,18 +1,26 @@
-import I18n from "i18n-js";
+import i18n from "i18next";
 import moment from "moment";
 import DateHelper from "./date";
+import "../../../__spec_helper__/I18next";
+
+i18n.addResourceBundle("en", "carbon", {
+  date: {
+    formats: {
+      inputs: ["DD/MM/YYYY"],
+    },
+  },
+});
+i18n.addResourceBundle("en-US", "carbon", {
+  date: {
+    formats: {
+      inputs: ["MM/DD/YYYY"],
+    },
+  },
+});
 
 describe("DateHelper", () => {
   beforeEach(() => {
-    I18n.translations = {
-      en: {
-        date: { formats: { inputs: ["DD/MM/YYYY"] } },
-      },
-
-      us: {
-        date: { formats: { inputs: ["MM/DD/YYYY"] } },
-      },
-    };
+    console.log(i18n.language);
```

Comment:
Is there supposed to be a console log in here?

Reply:
good point, removed.